content/docs/glossary/clawback.mdx

```diff
@@ -0,0 +1,297 @@
+---
+title: Clawbacks
+order:
+---
+
+import { CodeExample } from "components/CodeExample";
+import { Alert } from "components/Alert";
+
+Protocol 17 introduces operations that allow asset issuers to maintain tighter control over how their asset is distributed to the world. Specifically, it gives them power to **burn their asset** after a [payment][ex1] or [claimable balance][ex2] to another account, effectively removing it from the recipient's balance sheet:
+
+> The amount of the asset clawed back is burned and is not sent to any other address. The issuer may reissue the asset to the same account or to another account if the intent of the clawback is to move the asset to another account.
+
+This allows for things like regulatory enforcement, safety and control over certain assets, etc. You can refer to [CAP-35](https://github.com/stellar/stellar-protocol/blob/master/core/cap-0035.md) for more motivations or technical details behind these new features.
+
+
+## Relevant operations
+Configuring and performing a clawback involves several discrete steps:
+
+ - setting up your account to enable clawbacks (via [SetOptions](../start/list-of-operations.mdx#set-options)),
+ - establishing trustlines for a particular asset (via [ChangeTrust](../start/list-of-operations.mdx#change-trust)),
+ - making a payment with a particular asset (via [Payment](../start/list-of-operations.mdx#payment) or [Create Claimable Balance](../start/list-of-operations.mdx#create-claimable-balance)), and finally
+ - clawing back some or all of the asset (via [Clawback](../start/list-of-operations.mdx#clawback) or [Clawback Claimable Balance](../start/list-of-operations.mdx#clawback-claimable-balance)).
+
+### Setting Account Options
+In order to enable clawback, an account must first specify the `AUTH_CLAWBACK_ENABLED` flag on its account. This will cause *every* subsequent trustline established from this account to have its corresponding `TRUSTLINE_CLAWBACK_ENABLED_FLAG` set automatically.
```

Comment:
I would change `an account` to `the issuer account`. What you have here is pretty clear, but I wouldn't be surprised if some people think that account is referring to the account that owns the trustline.

Reply:
Good call. 8f9e286